### DISCUSSION TITLE
Local coordinates from pos

### DISCUSSION THREAD

**Discussion by dev-hydrogen** [Minestom/Minestom#1028] [Help] [answered]
I have a pos, including pitch and yaw, and i would like to subtract local coordinates from it, similar to the vanilla command `/teleport ^ ^-0.5 ^`
How can i achieve this?

**Comment by TheMode** [comment, marked as answer]
You can use `Argument.parse` to reuse the parsing logic of a command argument. The argument id is used as the string to parse.
```java
RelativeVec value = Argument.parse(ArgumentType.RelativeVec3("^ ^-0.5 ^"));
player.sendMessage("Position: " + value.from(playerPos));
```
